### PR TITLE
DOT-1438 Switch brand field to use vue-multi-select component

### PR DESCRIPTION
**Pull request by edwh**
As title.  Also adds a warning if you're creating a new brand.

### CODE REVIEW

File: resources/assets/js/app.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ require('./bootstrap-sortable.js');
 require('select2');
 require('slick-carousel');
 require('summernote');
+require('summernote-cleaner');
```
Comment (ngm) [inline resources/assets/js/app.js:13]: From other ticket

**Review by ngm — APPROVED** [review]
Code all looks good :+1: 

Minor point - we don't usually check in `public/js/app.js` unless on prod, do we?

File: resources/assets/js/app.js (hunk 1 above)
Comment (edwh) [reply resources/assets/js/app.js]: Ah yes.  Removed.

**Comment by edwh** [comment]
> Minor point - we don't usually check in `public/js/app.js` unless on prod, do we?

No - must have had a search and replace.  Backed out that file change for neatness.

**Pull request merged**